### ISSUE TITLE
cell phone viewing - can't scroll down on text

### ISSUE THREAD

**Issue by ClareCat21** [defjosiah/dowling-hart#17]
I know most people probably won't be doing this lesson plan on cell phones, but in the age of smart phones, I think it is important for the website to be viewable, at least

**Comment by wcaleb** [comment]
I agree, though I think making this totally mobile friendly will take a lot of work, so it's a long-term goal.

Which parts of the website don't scroll for you? All of them? Or just certain pages/sections?

**Comment by ClareCat21** [comment]
The floating part in the center, so the timeline text or the documents if they are longer than the initial view (which they almost  all are with pictures and stuff)

**Comment by wcaleb** [comment]
Hm. Works on my iPhone.  Do you have an Android device?

**Comment by ClareCat21** [comment]
Yes i have an Android. There is no scroll option in the grey box, but maybe there is another way to scroll and i just don't know how.
